### ISSUE TITLE
v2.3.2 iOS does not work

### ISSUE THREAD

**Issue by spring321** [tekartik/sqflite#1081]
![企业微信截图_259fdcdb-89e0-43ed-ac34-68d01db88843](https://github.com/tekartik/sqflite/assets/19162722/25bbe2e4-14a9-4bc6-8217-f2401c1e9fe8)

The iOS platform implementation is missing in v2.3.2, as shown in the figure

**Comment by alextekartik** [comment]
v2.3.2 uses the new darwin shared implementation as described [here](https://docs.flutter.dev/packages-and-plugins/developing-packages#shared-ios-and-macos-implementations). I don't know what you have tried and what kind of error you get. Can you try:
- making sure you are using the latest flutter
- flutter clean
- deleting Podfile.lock and Pods folder

And if that does not work:
- re-creating the ios project

Let me know if it works!

**Comment by jt274** [comment]
Related issue - When installing pods I get the error `No podspec found for 'sqflite' in '.symlinks/plugins/ios'`

I have:

- Deleted Podfile.lock, Pods folder, .symlinks folder
- flutter clean
- flutter pub get
- pod install

Same error. Flutter and cocoapods are latest version.

**Comment by alextekartik** [comment]
It seems that moving to the darwin (shared macos/ios) causes the issue, sorry. Does the issue happen on the example project or on a simple demo project?:
- https://github.com/tekartik/sqflite/tree/master/sqflite/example
- https://github.com/alextekartik/flutter_app_example/tree/master/demo_sqflite

In both I can simply
- create the project `flutter create --platforms ios .`
- start the ios simulator
- run `flutter run`

Sorry I cannot reproduce, ios project have always been a pain for me (I often have to re-create them)

**Comment by jt274** [comment]
> Sorry I cannot reproduce, ios project have always been a pain for me (I often have to re-create them)

Agreed, always a pain. This is an existing project, I am not sure how to fix it.

**Comment by davidmartos96** [comment]
What version of Flutter are you using? @jt274

**Comment by jt274** [comment]
> What version of Flutter are you using? @jt274

`3.16.9`

**Comment by jakcal** [comment]
I'm having the same issues here, I've just updated to Flutter V3.16.9 • channel stable.

**Comment by alextekartik** [comment]
@jt274 @jakcal do you have more information than just "does not work". What kind of error do you get?

Can you try the steps below in a console show me the output?

```
mkdir testiosapp
cd testiosapp
flutter create -t app --platforms ios .
flutter pub add sqflite
flutter build ios
```

If it fails, can you try `dart pub cache repair`and try the same steps?
